[libc][NFC] Make `EXP_MANT_MASK` an implementation detail

gchatelet opened the pull request:
This mask is an implementation detail of `FPBits` and shouldn't really leak outside of it.

legrosbuffle commented on libc/src/__support/FPUtil/FloatProperties.h:
```diff
@@ -162,13 +165,20 @@ struct FPProperties : public internal::FPBaseProperties<fp_type> {
+  //---------------------------------------------------------------------------
+
+  // Returns the absolute value of 'value'.
+  LIBC_INLINE static constexpr StorageType abs(StorageType value) {
```
It's weird to have a member function in a class named `FloatProperties`. This should probably be a free function with template properties.

Actually, I'm noting that this is aways called on the result of calling `FPBits::uintval()`, maybe it's a sign that this should be a member function of `FPBits` ?

gchatelet replied:
Fair enough, I wanted to unify `FloatProperties` and `FPBits` at some point but it's probably better to do this afterwards. It is clear that `FPBits` is a better place for `abs` right now. Thx for the suggestion.

gchatelet committed:
[libc][NFC] Make EXP_MANT_MASK an implementation detail

This mask is an implementation detail of `FPBits` and shouldn't really leak outside of it.
